api/config/config_prod.json

```diff
@@ -12,7 +12,7 @@
     "billingRetryCount": 4,
     "billingProjectBufferCapacity": 20,
     "xAppIdValue": "AoU-RW",
-    "vpcServicePerimeterName": ""
+    "vpcServicePerimeterName": "accessPolicies/357236995176/servicePerimeters/terra_prod_aou_prod"
```

Comment:
I guess we've already settled on this scheme, but I would've probably just had this field hold the name of the perimeter, e.g. `terra_prod_aou_prod` and filled in the policy value programmatically.